src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -839,6 +840,49 @@ def _corruptJobStore(self):
         assert isinstance(self.master, AWSJobStore)  # type hinting
         self.master.filesBucket.delete()
 
+    def testSDBDomainsDeletedOnFailedJobstoreBucketCreation(self):
+        """
+        This test ensures that SDB domains bound to a jobstore are deleted if the jobstore bucket
+        failed to be created.  We simulate a failed jobstore bucket creation by using a bucket in a
+        different region with the same name.
+
+        :returns: None
+        """
+        from boto.sdb import connect_to_region
+        from boto.s3.connection import Location, S3Connection
+        aws_west_regions = {'us-west-1': Location.USWest,
+                            'us-west-2': Location.USWest2}
+        externalAWSRegion = [R for r, R in aws_west_regions.items() if r != self.awsRegion()][0]
```

Comment:
`region, location` instead of `r, R` would probably be more readable. If line gets too long wrap before `if`